test/e2e/user_workload_monitoring_test.go

```diff
@@ -394,9 +349,9 @@ func assertMetricsForMonitoringComponents(t *testing.T) {
 	} {
 		t.Run(service, func(t *testing.T) {
 			f.ThanosQuerierClient.WaitForQueryReturn(
-				// To avoid making the test wait for more than lookback-delta in case Prometheus
-				// wasn't able to write stale markers (because it was down), reduce the lookup period.
-				t, time.Minute, fmt.Sprintf(`count(last_over_time(up{service="%s",namespace="openshift-user-workload-monitoring"}[1m]) == 1)`, service),
+				// To avoid having to make the test run for more than lookback-delta in case Prometheus
+				// wasn't able to write stale markers (because it was down), reduce the lookup period and the timeout.
+				t, 3*time.Minute, fmt.Sprintf(`count(last_over_time(up{service="%s",namespace="openshift-user-workload-monitoring"}[1m]) == 1)`, service),
```

Comment:
It was 5m before https://github.com/openshift/cluster-monitoring-operator/commit/c9359c604c46c98dc9871e39411746f2a27d582b
I've noticed the check has been more flaky recently, increasing the timeout to stabilize the CI, I'll investigate further when I have some time.